server/zoom/meeting.go

```diff
@@ -64,3 +65,59 @@ type Meeting struct {
 		AuthenticationName           string `json:"authentication_name"`
 	} `json:"settings"`
 }
+
+type CreateMeetingRequest struct {
+	Topic          string `json:"topic"`
+	Type           int    `json:"type"`
+	StartTime      string `json:"start_time,omitempty"`
+	Duration       int    `json:"duration,omitempty"`
+	ScheduleFor    string `json:"schedule_for,omitempty"`
+	Timezone       string `json:"timezone,omitempty"`
+	Password       string `json:"password"`
+	Agenda         string `json:"agenda"`
+	TrackingFields []struct {
+		Field string `json:"field"`
+		Value string `json:"value"`
+	} `json:"tracking_fields"`
+	// Recurrence struct {
+	// 	Type           int    `json:"type"`
+	// 	RepeatInterval int    `json:"repeat_interval"`
+	// 	WeeklyDays     string `json:"weekly_days,omitempty"`
+	// 	MonthlyDay     int    `json:"monthly_day,omitempty"`
+	// 	MonthlyWeekDay int    `json:"monthly_week_day,omitempty"`
+	// 	EndTimes       int    `json:"end_times,omitempty"`
+	// 	EndDateTime    int    `json:"end_date_time,omitempty"`
+	// } `json:"recurrence,omitempty"`
```

Comment:
Any reason we would want to have this commented out, rather than uncomment or remove this?

Reply:
I would like to keep it as reference. Right now removing the comments is not an option because it will send the "recurrence" field in the payload, and will create errors on zoom side.

Reply:
Can you do `*Recurrence` with `omitempty`?

Reply:
Indeed I can and I will. :sweat_smile: